winui/Rating/Precision.md

```diff
@@ -10,3 +10,86 @@ documentation: ug
 # Precision in WinUI Rating
 
 This section explains the precision features available in the WinUI Rating control.
+
```

Comment:
@Priyanka-Vijayasankar  Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.

Reply:
Changed the above description as mentioned above.